helm/cups/templates/networkpolicies.yaml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
wrap this file in an if statement to conditionally deploy all or none NetworkPolicies